### PR TITLE
add constructors; add routes method to users handler

### PR DESCRIPTION
**Pull request by mikenai**
some minor cleanup

### CODE REVIEW

File: internal/handlers/users.go [hunk 1]
```diff
@@ -23,7 +24,15 @@ type Users struct {
 }
 
 func NewUsers(us UsersService) Users {
-	return Users{}
+	return Users{user: us}
+}
+
+func (u Users) Routes() http.Handler {
```
Comment (admarc) [inline internal/handlers/users.go:33]: Is it consider to be a better practice to have routes configuration on handlers layers than directly in main.go ?

**Comment by admarc** [comment]
Acknowledged PR :)

**Comment by vitalii-tkachuk** [comment]
Ack

File: internal/handlers/users.go (hunk 1 above)
Comment (mikenai) [reply internal/handlers/users.go:33]: Well. The idea is that we only decide with URI and which method should be used on that level. Remember when we were doing a unit we couldn't set properly the method and URI? Now we can improve the test and actually test POST, GET, etc requests. Meanwhile, on the main level, we decide on which subroute to attach this package, for example on `/users`.

Comment (mikenai) [reply internal/handlers/users.go:33]: hm. I forget to add changes to the tests. I will do it in another PR

**Comment by wesol05** [comment]
ack

**Review by wesol05 — APPROVED** [review]

**Pull request merged**